tools/build/third_party/perfetto/perfetto.BUILD

```diff
@@ -156,6 +161,7 @@ cc_library(
         "@gapid//tools/build:windows": [],
         "@gapid//tools/build:darwin": [],
         "@gapid//tools/build:linux": [
+            "src/base/watchdog_posix.cc",
```

Comment:
remove, this should not be necessary anymore and should only be used by standalone builds.

Reply:
The timing layer links in perfetto to use the client API.  These were required to get everything to build.